### PR TITLE
NLP review cleaning and sentiment

### PR DESCRIPTION
**Pull request by jorgenhanssen**
I have not added visualization in this PR, but I need it for other modules.

### CODE REVIEW

File: src/modules/review_sentiments.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+from typing import Dict, Any
+from .base_module import BaseModule
+from data_loader import Data
```
Comment (AKissmehl) [inline src/modules/review_sentiments.py:3]: Should be the path, not ..data_loader -?

Comment (jorgenhanssen) [reply src/modules/review_sentiments.py:3]: The program is run from inside the src folder and it is importi0ng from the file `src/data_loader.py`.

Comment (jorgenhanssen) [reply src/modules/review_sentiments.py:3]: Ref the `make` command:
```make
run:
   @sh -c "cd src && python3 -m main"
```

**Comment by jorgenhanssen** [comment]
I have added [Swifter](https://github.com/jmcarpenter2/swifter) to speed up the execution of pandas apply. So that dependency is needed

**Comment by jorgenhanssen** [comment]
There are some improvements to this code but its implement in #5

**Pull request merged**